tests/store/test_abstract_store.py

```diff
@@ -0,0 +1,86 @@
+import mock
+
+from mlflow.store.abstract_store import AbstractStore
+from mlflow.entities import ViewType
+
+
+class ConcreteStore(AbstractStore):
```

Comment:
🤣 that name.

Can we rename this to something like `TestAbstractStoreImpl`

Reply:
Done - though I do miss the `ConcreteStore` 😁 

I made it `AbstractStoreTestImpl` as classes named `Test*` are inspected by pytest for test methods, I believe.